controllers/reconcile_fake_test.go

```diff
@@ -0,0 +1,286 @@
+package controllers_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	tlog "github.com/go-logr/logr/testing"
+	"github.com/google/go-cmp/cmp"
+	konsumeratorv1alpha1 "github.com/lwolf/konsumerator/api/v1alpha1"
+	"github.com/lwolf/konsumerator/controllers"
+	"github.com/lwolf/konsumerator/pkg/helpers"
+	"github.com/lwolf/konsumerator/pkg/helpers/tests"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func initReconciler(consumer *konsumeratorv1alpha1.Consumer) (client.Client, *controllers.ConsumerReconciler) {
+	var s = runtime.NewScheme()
+
+	_ = clientgoscheme.AddToScheme(s)
```

Comment:
why omitting errors?

Reply:
honestly, have no idea. all the docs, including examples in the source code omit errors during schema registration.